02Lesson/02Project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
-</head>
+  <head>
+  <meta charset="utf-8">
+    <title>Daily Planet Newspaper</title>
+  <link href="css/style.css" rel="stylesheet"/>
+  </head>
+
 <body>
-  <!-- You code start here -->
+
+  <h1>Daily Planet</h1>
+  <nav>
+      <span><a href="#">Sports</a></span>
+      <span><a href="#">Weather</a></span>
+      <span><a href="#">News</a></span>
+  </nav>
+
+<main>
+  <section>
+      <article>
+        <img src="http://l.yimg.com/ny/api/res/1.2/B62CXNsaQ35mTkoif_ioXw--/YXBwaWQ9aGlnaGxhbmRlcjtzbT0xO3c9NzQ0O2g9NTY5/http://media.zenfs.com/en/homerun/feed_manager_auto_publish_494/899c290f5812d922a367b9aa90153016" alt="Dak">
+        <h4>With Dak Prescott leading the Cowboys to a 5-1 start on the season, why is Jerry Jones
+        thinking about replacing him with Tony Romo?
+        </h4>
+  
+        <p>Dak Prescott is the first rookie in NFL history to start his first 6 games without throwing an interception.  Furthermore, he has the Cowboys off to their best start in over 20 years.  Fans are already comparing this scenario to when Tom Brady took over for Drew Bledsoe in 2001...except for Jerry Jones.  Jones told the media on Sunday that Romo is the Cowboys quarterback, and it will stay that way once he is ready to return to the field.  However, many are wondering what Jones is thinking.  You don't ruin a good thing, and bringing Romo back is nowhere close to a gaurantee of a better team.  Garrett mentioned that he is not complaining about the situation at all, and is waiting to see it all play out, and that he is fortunate to have 2 top tier qb's in his pocket.
+
+        </p>
+      </article>
+    </section>
+  <aside>
+      <img src="http://thecityinfluencer.com/wp-content/uploads/2015/08/wpid-Dallas-cowboys-star-logo.jpg" alt="cowboys"/>
+  </aside>
+</main>
+    <footer>

```

Comment:
Here you can see that something with your tags is not right. After closing the `main` tag, there cannot be an element that is further indented than the closing tag. So either you missed some tags or the indentation is wrong somewhere...